cmd/webhook/main.go

```diff
@@ -63,8 +63,8 @@ func main() {
 		flag.StringVar(&opt.masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	}
 	flag.StringVar(&opt.admissionListenAddress, "listen-address", ":9876", "The address on which the MutatingWebhook will listen on")
-	flag.StringVar(&opt.admissionTLSCertPath, "tls-cert-path", "/tmp/cert/cert.pem", "The path of the TLS cert for the MutatingWebhook")
-	flag.StringVar(&opt.admissionTLSKeyPath, "tls-key-path", "/tmp/cert/key.pem", "The path of the TLS key for the MutatingWebhook")
+	flag.StringVar(&opt.admissionTLSCertPath, "tls-cert-path", "/tmp/cert/tls.crt", "The path of the TLS cert for the MutatingWebhook")
+	flag.StringVar(&opt.admissionTLSKeyPath, "tls-key-path", "/tmp/cert/tls.key", "The path of the TLS key for the MutatingWebhook")
```

Comment:
As both kubeone and KKP rely on default names:
* this will break kubeone https://github.com/kubermatic/kubeone/blob/2fd69d0e9431877bea5bb33b06bd95e287b17012/addons/machinecontroller/webhook.yaml#L6-L10

* and KKP https://github.com/kubermatic/kubermatic/blob/9aad988df4c621558ada17946f4425ca3e3097b6/pkg/resources/resources.go#L164-L167

Reply:
The idea is to use this in machine-controller and update this in KKP and KubeOne instead. Since cert-manager generates `tls.crt` and `tls.key`. We need to treat machine-controller as a standalone tool in this case.

Reply:
Sure, I meant to point out this, to not forget to update those filenames. Other approach could be — we shell use flags explicitly in the webhook deployment.